homeassistant/components/playstation_network/coordinator.py

```diff
@@ -44,15 +39,16 @@ def __init__(self, hass: HomeAssistant, api, user, client) -> None:
             update_interval=DEVICE_SCAN_INTERVAL,
         )
 
-        self.api = api
         self.user = user
-        self.client = client
-        self.psn = PlaystationNetworkData({}, "", {}, {}, {}, False, False, [], "", "")
+        self.psn: PlaystationNetworkData = PlaystationNetworkData(
+            {}, "", "", False, {}, {}
+        )
```

Comment:
No need to declare this here. Just gather all information within `_async_update_data` in local variables and create a `PlaystationNetworkData` at the end.